### PR TITLE
2022-11 LWG Motion 2

### PR DESCRIPTION
**Pull request by tkoeppe**

### CODE REVIEW

**Comment by tkoeppe** [comment]
@jwakely Care to perform a final review?

File: func.html [hunk 1]
```diff
@@ -85,64 +78,70 @@ <h1>Overview</h1>
...
+          the allocator is initialized from the second parameter.
+        </li>
+        <li>
+          For all other constructors, the allocator is value-initialized.
+        </li>
+      </ul>
       <p>
-        In the following descriptions, let <code><em>ALLOCATOR_OF</em>(f)</code> be the allocator specified in the construction of <code>function</code> <code>f</code>,
-        or the value of <code>experimental::<wbr>pmr::<wbr>get_default_resource()</code> at the time of the construction of <code>f</code> if no allocator was specified.
+        In all cases, the allocator of a parameter having type <code>function&&</code> is unchanged.
```
Comment (jwakely) [inline func.html]:
```suggestion
        In all cases, the allocator of a parameter having type <code>function&amp;&amp;</code> is unchanged.
```

Is this needed?

File: func.html [hunk 2]
```diff
@@ -85,64 +78,70 @@ <h1>Overview</h1>
...
       <cxx-function>
         <cxx-signature>function&amp; operator=(nullptr_t) noexcept;</cxx-signature>
 
         <cxx-effects>If <code>*this != nullptr</code>, destroys the target of <code>this</code>.</cxx-effects>
-        <cxx-postconditions><code>!(*this)</code>.
-        The memory resource returned by <code>get_memory_resource()</code> after the assignment is equivalent to the memory resource before the assignment.
-        <cxx-note>the address returned by <code>get_memory_resource()</code> might change</cxx-note></cxx-postconditions>
+        <cxx-postconditions>
+          <code>!(*this)</code>.
+          <cxx-note>the stored allocator is unchanged</cxx-note>
```
Comment (jwakely) [inline func.html]:
```suggestion
          <cxx-note>The stored allocator is unchanged.</cxx-note>
```

File: func.html (hunk 1 above)
Comment (tkoeppe) [reply func.html]: No, it doesn't actually seem necessary, but it's still polite, so I'll accept it!

File: func.html (hunk 2 above)
Comment (tkoeppe) [reply func.html]: Applied, thanks!

**Review by jwakely — APPROVED** [review]

**Pull request merged**